### ISSUE TITLE
4.1.3 - Help for Search (Community portal)

### ISSUE THREAD

**Issue by janahoffmann** [gbhl/bhl-europe#106]
Provide link to help and tutorial.

Precondition: 
Create help/ tutorial for search.

**Comment by akohlbecker** [comment]
A couple of  questions in order to clarify what actually should be done:
- Where should this link be placed, next to any search form?
- Since we have different searches: simple, advanced, we would need different help texts. All help texts could be on the same web page but marked up with specific anchor tags. so that it is possible to "jump" to the according paragraph using URI fragments (e.g: help/search#advanced, help/search#simple)
- The Help/ tutorial for is to be written in the BHLWiki or on the project page. This text is then is transferred into a Drupal node, right? I suppose writing this text is another task, so we should open another ticket for it.

**Comment by JiriFrank** [comment]
Yes Andreas, I agree. There are two separated task. The first task is preparing the help text and publishing it on the project website or Wiki. The second task is implementation of this help on the portal.

From my view is the most complicated thing to prepare the text ;-) I can after published this text on the project website in specific portal menu and create the structure using anchors etc., it is not a problem.

For the second task there are several solutions:

The attractive way is to implement in all blocs with specific function a small question mark. When you click on it, the specific help for that feature could appear in lightbox view.  There could be also the link for more information, which will be published on the project website, connected with anchors etc..

The second solution, easier but less attractive, is implement the help link close to the features, which will be directly connected with the published text on the project website and appeare in new browser window. The good thing is, the we are keeping the same corporate design for the portal and project website, so user will be still in BHL-Europe environment. 

Both solutions are possible and Drupal doesn't have problem with that. But the most important thing what we need to know before we start writing the text is the specific real functionality of all features developed and implemented on the website. We can start prepare the text step by step with the functional features on the portal. 

@akohlbecker
@janahoffmann

This issue is closely connected with issues #107, #108 and could provide the same solution.

**Comment by JiriFrank** [comment]
Help/Tutorial for simple and advanced search is done same for Browse. The tutorial will be integrated to the portal as a combination of hint bar and tutorial links to the lightbox view on each page. Texts are wainting for corrections and translations.

**Comment by JiriFrank** [comment]
4.1.3 - Help for Search (Community portal)

COR number: 4.1.1
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Portal Front page and Advanced search

Description:

Provide link to help and tutorial.

Several parts of portal are still under developing and will be changed in some details, this will also affect the final version of tutorial. Help is provided as tutorial and hint bar.

@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

**Comment by JiriFrank** [comment]
4.1.3 - Help for Search (Community portal)

COR number: 4.1.1
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Portal Front page and Advanced search

Description:

Provide link to help and tutorial.

Several parts of portal are still under developing and will be changed in some details, this will also affect the final version of tutorial. Help is provided as tutorial and hint bar.

@AntonioGVH
@fwelter
@LarissaS

**Comment by LarissaS** [comment]
Tutorial works, but have seen one bug:
if you want to go to search page from news page you come to advanced search and not simple search. And results of former searching are still hanging on the page.

-----Original Message-----

**Comment by HenningScholz** [comment]
Some tutorials are working, although the text needs some revisions, but this will probably come when the portal will be finalised.

---

Von: JiriFrank [mailto:reply@reply.github.com]
Gesendet: Di 17.01.2012 15:28
An: Scholz, Henning
Betreff: Re: [bhle] 4.1.3 - Help for Search (Community portal) (#106)

4.1.3 - Help for Search (Community portal)

COR number: 4.1.1
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Portal Front page and Advanced search

Description:

Provide link to help and tutorial.

Several parts of portal are still under developing and will be changed in some details, this will also affect the final version of tutorial. Help is provided as tutorial and hint bar.

@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

---

**Comment by RalfH** [comment]
Generally, these functionalities are good. Texts not too long.

Tutorial Advanced search: Green colour and white text are not really pleasing the eyes. The box seems a bit too wide with very long lines with rather small letters.

Tutorial Simple search: 10 line text block should be more subdivided in paragraphs or so.

-----Ursprüngliche Nachricht-----
Von: JiriFrank [mailto:reply@reply.github.com] 
Gesendet: Dienstag, 17. Januar 2012 15:28
An: Hand, Ralf
Betreff: Re: [bhle] 4.1.3 - Help for Search (Community portal) (#106)

4.1.3 - Help for Search (Community portal)

COR number: 4.1.1
Testing platform: http://bhl-test.nhm.ac.uk/portal/
Function: Portal Front page and Advanced search

Description:

Provide link to help and tutorial.

Several parts of portal are still under developing and will be changed in some details, this will also affect the final version of tutorial. Help is provided as tutorial and hint bar.

@AnneSch
@AntonioGVH
@fwelter
@grahamhrbge1670
@heimor
@HenningScholz
@JFTester
@JiriFrank
@LarissaS
@RalfH

---

**Comment by grahamhrbge1670** [comment]
Links to Tutorials work well. Explanatory text along base of home page perhaps needs a bit more emphasis.

**Comment by AntonioGVH** [comment]
There is no a tutorial in the proper  sense, but different screen are explained in the tutorial individually.

Some parts need to be translated to Spanish

Quoting JiriFrank:

**Comment by JiriFrank** [comment]
Helps system overall works.
Some modifications need to be done.
Text for helps will be finalized in last phase of portal developing according to final portal functionalities.

My task :-)
